StripeFinancialConnections/StripeFinancialConnections/Source/Native/PartnerAuth/PartnerAuthViewController.swift

```diff
@@ -25,7 +25,18 @@ protocol PartnerAuthViewControllerDelegate: AnyObject {
 
 @available(iOSApplicationExtension, unavailable)
 final class PartnerAuthViewController: UIViewController {
-    
+
+    /**
+     Unfortunately there is a need for this state-full parameter. When we get url callback the app might not be in foreground state.
+     If we then authorize the auth session will fail as you can't do background networking without special permission.
+     */
+    private var unprocessedReturnURL: URL?
```

Comment:
now that we have this in two places, it would be interesting if there's a way to create a "container/handler" for some of this logic to "tuck" it away, but that's a polish thing